Kontent.Ai.Management/Modules/ResiliencePolicy/DefaultResiliencePolicyProvider.cs

```diff
@@ -13,16 +14,6 @@ namespace Kontent.Ai.Management.Modules.ResiliencePolicy;
 /// </summary>
 public class DefaultResiliencePolicyProvider : IResiliencePolicyProvider
 {
-    private static readonly HttpStatusCode[] StatusCodesToRetry =
```

Comment:
I think it might be somewhere in the docs (MD files).

Reply:
Well now we support the following:

-         RequestTimeout = 408,
-         InternalServerError = 500,
-         NotImplemented = 501,
-         BadGateway = 502,
-         ServiceUnavailable = 503,
-         GatewayTimeout = 504,
-         HttpVersionNotSupported = 505,
-         VariantAlsoNegotiates = 506,
-         InsufficientStorage = 507,
-         LoopDetected = 508,
-         NotExtended = 510,
-         NetworkAuthenticationRequired = 511

Is it ok?

Reply:
Let'S sync about that - this is the list from API reference:
https://kontent.ai/learn/reference/openapi/delivery-api/#section/HTTP-status-codes-summary

@Enngage - which ones have we covered in MAPI/DAPI JS SDK?

Reply:
OK let't try to retry 4XX and 5XX Status codes, but out Kontent Errors containing specific info would don't retry: https://kontent.ai/learn/reference/openapi/delivery-api/#section/Resolving-errors

Once ready add
=> add to https://kontent-ai.github.io/Guidelines-for-SDK-developers information
=> submit issue to core-js-sdk